app/views/schools/choose_programme/confirm_programme.html.erb

```diff
@@ -9,11 +9,7 @@
     <p class="govuk-body">You have chosen to:</p>
 
     <ul class="govuk-list govuk-list--bullet">
-      <% if @induction_choice_form.programme_choice == "full_induction_programme" %>
-        <li>Use a training provider, funded by the DfE</li>
-      <% else %>
-        <li>Deliver your own programme using DfE accredited materials</li>
-      <% end %>
+      <li><%= @induction_choice_form.programme_choice_words %></li>
```

Comment:
Is this the same? I think the programme_choice have `(full induction programme)` and `(core induction programme)` in them in addition to those options.

Reply:
And percy noticed that :O 
Also, looks like cypress tests are broken 😢

Reply:
Yup, there are small changes. I was hoping those are all right given that FIP and CIP are a technical jargon. Content files are not too helpfull there (I don't know what does it mean when the text in brackets is stroke orange?) and prototype doesn't show these neither.

Cypress fixed.